### DISCUSSION TITLE
ACF Extended Support

### DISCUSSION THREAD

**Discussion by jenishshrestha** [wp-graphql/wp-graphql#1736] [Q&A]
Any plans on supporting [acf extended](https://wordpress.org/plugins/acf-extended/) plugin?

**Comment by jasonbahl** [comment]
👋  No plans to support ACF extensions out of the box, as there are so many it would be quite difficult to maintain compatibility with them all. 

But @markkelnar and I are working on documentation for how ACF extensions can make use of filters within WPGraphQL and WPGraphQL for ACF to allow extensions to provide their own WPGraphQL support, or at least for project maintainers to hook in with the support on their own. 

Keep an eye out for updates in this regard!
